package.json

```diff
@@ -229,7 +229,7 @@
     "unist-util-visit": "^2.0.3",
     "uuid": "^8.3.2",
     "vast-client": "^3.1.1",
-    "video.js": "^7.13.3",
+    "video.js": "git://github.com/mayeaux/video-js-fork",
     "videojs-contrib-quality-levels": "^2.0.9",
```

Comment:
### Stopping of streams not handled well

- On both mobile and desktop layout, it goes into perpetual spinning mode.
- On mobile, the alignment is off.


<img width="441" alt="image" src="https://user-images.githubusercontent.com/64950861/155056856-0ff7a345-2272-4b90-838e-9eaee5b63b89.png">

<img width="846" alt="image" src="https://user-images.githubusercontent.com/64950861/155056958-4b933636-d9d2-459b-9a42-8a9b81a37838.png">

Reply:
This is fixed on other branch